boards/arm/nrf7002dk_nrf5340/nrf5340_cpuapp_common.dts

```diff
@@ -95,6 +95,15 @@
 		};
 	};
 
+	nrf_radio_coex: nrf7002-coex {
+		status = "okay";
+		compatible = "nordic,nrf700x-coex";
+		req-gpios = <&gpio0 30 GPIO_ACTIVE_HIGH>;
+		pri-dir-gpios = <&gpio0 28 GPIO_ACTIVE_HIGH>;
```

Comment:
```suggestion
		req-gpios = <&gpio0 28 GPIO_ACTIVE_HIGH>;
		pri-dir-gpios = <&gpio0 30 GPIO_ACTIVE_HIGH>;
```
This seems to be a bug in the original DTS file. Please could you correct this in this PR for this and other relevant files.

Reply:
Done

Reply:
Thanks!